README.md

```diff
@@ -50,20 +50,6 @@ _The Documentation matches the most recent Nuget release from the `main` branch
 descriptors. Most classes are safe for threading.
 * **Reactive Extensions** - Use [reactive extensions](https://github.com/dotnet/reactive) and benefit from increased code readability, and the ability to apply the MVVM pattern and [ReactiveUI](https://www.reactiveui.net/) data bindings. See the [source code](https://github.com/gui-cs/Terminal.Gui/tree/master/ReactiveExample) of a sample app in order to learn how to achieve this.
 
-## Showcase & Examples
```

Comment:
There's no link to showcase.md in readme.md. Please leave this header in, and provide a link.

Reply:
@En3Tho can you please address this?

Reply:
@tig Thank you for you patience. Please take a look